.travis.yml

```diff
@@ -3,20 +3,21 @@ language: python
 python:
   - "2.7"
 
+env:
+  - LINT_CHECK="1"
+  - VERSION="8.0" ODOO_REPO="odoo/odoo" LINT_CHECK="0"
+  - VERSION="8.0" ODOO_REPO="OCA/OCB" LINT_CHECK="0"
+  - VERSION="8.0" ODOO_REPO="odoo/odoo" UNIT_TEST="1" LINT_CHECK="0"
```

Comment:
The template is a bit outdated I think, because we have put that and Travis delas to complete sometimes an hour! That's why we remove this line.